src/main/webapp/app/shared/pipes/filterfeedbacks.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { PipeTransform, Pipe } from '@angular/core';
+import { Feedback } from 'app/entities/feedback';
+
+@Pipe({ name: 'filterfeedbacks' })
+export class FilterFeedbacksPipe implements PipeTransform {
+    transform(feedbacks: Feedback[]): any {
+        if (!feedbacks) {
```

Comment:
This check seems strange to me - it shouldn't be possible that feedbacks is null or undefined.
Also this would cause an issue in ngFor. It would be better to make sure that the component doesn't create a null/undefined value for feedbackList.

Reply:
I've removed the check - I thought it was needed as your removekeys pipe also had it, but its not.